src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8735,6 +8742,7 @@ namespace UnitarySystems {
             m_initLoadBasedControlAirLoopPass = false;
         }
         if (!DataGlobals::BeginEnvrnFlag) {
+            this->m_MyEnvrnFlag2 = true; // this does not appear to be needed, only initializes autosized coil fluid flow rates
```

Comment:
Just moved this from one location to another, no need to test !DataGlobals::BeginEnvrnFlag twice.